tests/unit_tests/api/test_server.py

```
@@ -0,0 +1,166 @@
+
+
+class TestBuildAgentArgs:
+    def test_maps_fields_correctly(self):
+        args = argparse.Namespace(
+            namespace="myns",
+            config="/etc/conf.yml",
+            max_steps=15,
+            workflow="fixed",
+            load_cp=None,
+            debug=True,
+            host="0.0.0.0",
```

Ruff (0.15.6)

[error] Possible binding to all interfaces

(S104)